### ISSUE TITLE
Cloud run region warning

### ISSUE THREAD

**Issue by ghost** [MarkEdmondson1234/googleCloudRunner#192]
I get the following warning after an otherwise successful build of the demo R shiny app. I'm hoping it's just a case of a stale check given the significantly greater regions available on the Google Cloud Run service these days. That minor bug aside thanks for an amazing package with great documentation. 

`==CloudRunService==
name:  shiny-cloudrun 
location:  australia-southeast2 
lastModifier:  612593503702@cloudbuild.gserviceaccount.com 
containers:  gcr.io/clound-run-demo/shiny-cloudrun:357a191b-7f02-49fd-877f-d81675b565a9 
creationTimestamp:  2022-05-13T04:26:07.842188Z 
observedGeneration:  1 
Warning message:
In make_endpoint(sprintf("namespaces/%s/services/%s", projectId,  :
  Endpoint is not one of us-central1 asia-northeast1 europe-west1 us-east1 got: australia-southeast2
> cr_regions
 [1] "asia-east1"              "asia-east2"              "asia-northeast1"         "asia-northeast2"         "asia-northeast3"        
 [6] "asia-south1"             "asia-south2"             "asia-southeast1"         "asia-southeast2"         "australia-southeast1"   
[11] "australia-southeast2"    "europe-central2"         "europe-north1"           "europe-west1"            "europe-west2"           
[16] "europe-west3"            "europe-west4"            "europe-west6"            "northamerica-northeast1" "northamerica-northeast2"
[21] "southamerica-east1"      "us-central1"             "us-east1"                "us-east4"                "us-west1"               
[26] "us-west2"                "us-west3"                "us-west4"    `

**Comment by MarkEdmondson1234** [comment]
Yep you are right, the region list just needs updating. Thanks for flagging it.
